scripts/newinstall.sh

```diff
@@ -217,6 +214,43 @@ if true; then
 	fi
 fi
 
+##########	Test/warn about the existence of ~/.eups
+
+if true; then
+	if [[ -e ~/.eups ]]; then
+		TIMESTAMP=$(date +%Y%m%d%S)
+		cat <<-EOF
+
+		Problems have been observed when re-installing or upgrading EUPS while
+		preserving a pre-existing ~/.eups directory.  This directory primarily
+		contains cache data and is safe to remove when EUPS is not actively
```

Comment:
I don't know what problems you are having.  I'd much rather fix the problem than remove the files.

As always, I'll want to know if the problem is in eups or the distribution system.